### ISSUE TITLE
Selected State does not annonce when TextInput Component selected

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30955]
## Description
Does not announce "selected" on the focus of TextInput Component.

This issue is covered in-depth by parent issue #30842, please check there for more details.

## React Native version:
0.63

**Comment by fabriziobertoglio1987** [comment]
Hi @Naturalclar

I saw you thumbs up this issue :+1: justed wanted to tell you that I found a solution and I prepared a pr https://github.com/facebook/react-native/pull/31144
The pr is in draft and still in progress :construction: 

Thanks a lot :pray: :peace_symbol:

**Comment by fabriziobertoglio1987** [comment]
Hello @amarlette 

thanks a lot for this issue. I wrote pr https://github.com/facebook/react-native/issues/30955 that fixes this issue on Android TalkBack. I will now move to the next issue in https://github.com/facebook/react-native/projects/15

I wish you a good day :peace_symbol: :pray: 
I remain available for improvements
Thanks

**Comment by amarlette** [comment]
Thank you @fabriziobertoglio1987 for your contribution! I look forward to seeing the next issue you tackle. 

Also to clarify I opened these on behalf of the React Native core team so they will be the ones to review your PR #31144 but it is exciting to see work being done on these issues.
